### PR TITLE
EQPS-1001: EQP MFTF Update Devdocs

### PR DESCRIPTION
**Pull request by magterskine**
## Purpose of this pull request

- Update to explain that tests do not affect Technical Review status
- Includes list of test executed for MFTF Magento Supplied test

## Affected DevDocs pages

https://devdocs.magento.com/marketplace/sellers/mftf-magento.html

https://devdocs.magento.com/marketplace/sellers/mftf-vendor.html

### CODE REVIEW

File: src/marketplace/sellers/mftf-magento.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ title: MFTF Magento-supplied Tests
 
 The Magento Functional Testing Framework (MFTF) is a browser-based acceptance testing framework used to validate the functionality of a Magento site. Running Magento-supplied MFTF tests aids Extension Quality Program (EQP) QA efforts in assessing critical functionality of a Magento instance with the extension installed.
 
+Magento-Supplied MFTF tests are not required to pass for technical review. The outcome of this testing is used to inform QA efforts and provide feedback on functional quality.
```
Comment (BarnyShergold) [inline src/marketplace/sellers/mftf-magento.md:10]:
```suggestion
Magento supplied MFTF tests are not required to pass for technical review. The outcome of this testing is used to assist with QA efforts and provide feedback on functional quality.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Nice clarification - just a few corrections

**Comment by magterskine** [comment]
@BarnyShergold Thanks for the review and suggestion - made the changes requested

File: src/marketplace/sellers/mftf-vendor.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ title: MFTF Vendor-supplied Tests
 
 The Magento Functional Testing Framework (MFTF) is a browser-based acceptance testing framework used to validate the functionality of a Magento site. Running vendor-supplied MFTF tests helps to ensure that the extension functionality is operating as expected for the end user.
 
+Vendor-Supplied MFTF tests are not required to pass for technical review. The outcome of this testing is used to inform QA efforts and provide feedback on functional quality.
```
Comment (BarnyShergold) [inline src/marketplace/sellers/mftf-vendor.md:10]:
```suggestion
Vendor supplied MFTF tests are not required to pass for technical review. The outcome of this testing is used to assist with QA efforts and provide feedback on functional quality.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Very strange -  I made this correction the first time round but it has disappeared!

**Review by BarnyShergold — APPROVED** [review]
👍

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8829`.

**Pull request merged**